pkgs/games/multimc/default.nix

```
@@ -33,7 +33,16 @@ in mkDerivation rec {
       Allows you to have multiple, separate instances of Minecraft (each with their own mods, texture packs, saves, etc) and helps you manage them and their associated options with a simple interface.
     '';
     platforms = platforms.linux;
-    license = licenses.lgpl21Plus;
+    license = with licenses; [
+      # Per https://github.com/MultiMC/MultiMC5/blob/develop/COPYING.md
+      asl20
+      lgpl21Plus
+      lgpl3Plus
```

```suggestion
      lgpl3Plus
      gpl2Classpath
```
As I mentioned, true until the next version bump

---

Also this is a claim based on the [COPYING](https://github.com/MultiMC/MultiMC5/blob/0.6.11/COPYING.md#pack200) file for that version not on my own analysis.